sonic-thermalctld/tests/test_thermalctld.py

```diff
@@ -133,6 +148,35 @@ def test_fanupdater_fan_over_speed():
     logger.log_notice.assert_called_once()
 
 
+def test_insufficient_fan_number():
+    fan_status1 = FanStatus()
+    fan_status2 = FanStatus()
+    fan_status1.set_presence(False)
+    fan_status2.set_fault_status(False)
+    assert FanStatus.get_bad_fan_count() == 2
+    FanStatus.reset_fan_counter()
+    assert FanStatus.get_bad_fan_count() == 0
+
+    chassis = MockChassis()
+    chassis.make_absence_fan()
+    chassis.make_fault_fan()
+    fan_updater = FanUpdater(chassis)
+    fan_updater.update()
+    assert logger.log_warning.call_count == 3
+    logger.log_warning.assert_called_with('Insufficient number of working fans warning: 2 fans are not working.')
+
+    fan_list = chassis.get_all_fans()
+    fan_list[0].presence = True
+    fan_updater.update()
+    logger.log_warning.assert_called_with('Insufficient number of working fans warning: 1 fans are not working.')
+    assert logger.log_notice.call_count == 1
```

Comment:
While there is no functional difference, I suggest swapping lines 171 and 172 to make the order of operations consistent with the stanzas above and below.

Reply:
Fixed